interactive/views.py

```diff
@@ -43,6 +43,7 @@ def build_analysis(self, *, form_data, project):
             created_by=self.request.user.email,
             demographics=raw.get("demographics", ""),
             filter_population=raw.get("filterPopulation", ""),
+            purpose=raw.get("purpose", ""),
```

Comment:
I'm not sure why, but I am not able to submit an analysis request because I'm not pulling in the updated schema from interactive-templates. If it works for you locally though, I'm going to assume there's something weird in my local setup.

Reply:
I had to run `pip uninstall interactive_templates && touch requirements.prod.in && just devenv` to get this working and have a note to see if I can work out why.  I should have mentioned on this PR, sorry!